plugins/uefi-dbx/fu-uefi-dbx-common.c

```diff
@@ -89,9 +89,16 @@ fu_uefi_dbx_signature_list_validate(FuContext *ctx, FuEfiSignatureList *siglist,
 	for (guint i = 0; i < volumes->len; i++) {
 		FuVolume *esp = g_ptr_array_index(volumes, i);
 		g_autoptr(FuDeviceLocker) locker = NULL;
-		locker = fu_volume_locker(esp, error);
-		if (locker == NULL)
-			return FALSE;
+		g_autoptr(GError) error_local = NULL;
+		locker = fu_volume_locker(esp, &error_local);
+		if (locker == NULL) {
+			if (!g_error_matches(error_local, G_IO_ERROR, G_IO_ERROR_NOT_SUPPORTED)) {
+				g_propagate_error(error, g_steal_pointer(&error_local));
+				return FALSE;
+			}
+			g_debug("failed to mount ESP: %s", error_local->message);
```

Comment:
This is a pretty uncommon situation; shouldn't this be a warning?

Reply:
Yup, agreed. I've upgraded to a g_warning().